docs/lib-documentation.md

```diff
@@ -120,21 +120,46 @@ Repartitions the network so that the connections between services match the spec
-Waits until a service endpoint is available by making requests to the endpoint using the given parameters. An error is thrown if the number of retries is exceeded.
+### waitForEndpointAvailabilityHttpGet(ServiceID serviceId, uint32 port, String path, String requestBody, uint32 initialDelayMilliseconds, uint32 retries, uint32 retriesDelayMilliseconds, String bodyText)
+Waits until a service endpoint is available by making requests to the endpoint using the given parameters, and the http post method. An error is thrown if the number of retries is exceeded.
+
+**Args**
+
+* `serviceId`: The ID of the service to check.
+* `port`: The port (e.g. 8080) of the endpoint to check.
+* `path`: The path of the service to check, which must not start with a slash (e.g. `service/health`).
+* `initialDelayMilliseconds`: Number of milliseconds to wait until executing the first HTTP call
+* `retries`: Max number of HTTP call attempts that this will execute until giving up and returning an error
+* `retriesDelayMilliseconds`: Number of milliseconds to wait between retries
+* `bodyText`: If this value is non-empty, the endpoint will not be marked as available until this value is returned (e.g. `Hello World`). If this value is emptystring, no body text comparison will be done.
+
+### waitForEndpointAvailabilityHttpPost(ServiceID serviceId, uint32 port, String path, String requestBody, uint32 initialDelayMilliseconds, uint32 retries, uint32 retriesDelayMilliseconds, String bodyText)
+Waits until a service endpoint is available by making requests to the endpoint using the given parameters, and the http get method. An error is thrown if the number of retries is exceeded.
 
 **Args**
 
 * `serviceId`: The ID of the service to check.
-* `httpMethod`: The HTTP method of the endpoint to check. Allowed values: GET or POST
 * `port`: The port (e.g. 8080) of the endpoint to check.
 * `path`: The path of the service to check, which must not start with a slash (e.g. `service/health`).
 * `requestBody`: The request body content that will be sent to the endpoint being checked.
-* `initialDelaySeconds`: Number of seconds to wait until executing the first HTTP call
+* `initialDelayMilliseconds`: Number of milliseconds to wait until executing the first HTTP call
 * `retries`: Max number of HTTP call attempts that this will execute until giving up and returning an error
 * `retriesDelayMilliseconds`: Number of milliseconds to wait between retries
 * `bodyText`: If this value is non-empty, the endpoint will not be marked as available until this value is returned (e.g. `Hello World`). If this value is emptystring, no body text comparison will be done.
 
+### getServices() -\> Set servicesID
+
+**Returns**
+
+* `servicesID`: A set of service's ID 
+
+### getLambdas() -\> Set lambdasID
+
```

Comment:
Would recommend the same sort of quick blurb - "Gets the IDs of the Kurtosis Lambdas that have been loaded into the test network."